code/modules/jobs/job_types/security_officer.dm

```diff
@@ -231,6 +231,22 @@ GLOBAL_LIST_INIT(available_depts, list(SEC_DEPT_ENGINEERING, SEC_DEPT_MEDICAL, S
 	courierbag = /obj/item/storage/backpack/messenger/inteq
 	backpack_contents = list(/obj/item/melee/baton/loaded=1, /obj/item/ammo_box/magazine/co9mm=1, /obj/item/gun_voucher=1,/obj/item/pda/security)
 
+/datum/outfit/job/security/inteq/beluga
+	name = "IRMG Enforcer (Inteq)"
```

Comment:
You may want to include a (Beluga) here, as I believe the enforcer is already defined with this name elsewhere.

Reply:
donezo